### PR TITLE
Adds ability to exclude data and tables from dumps

### LINKED ISSUE #1: Add ability to exclude data for specified tables

### LINKED ISSUE #4: Add ability to exclude entire tables

### PR DESCRIPTION
**Pull request by nicksantamaria**
Fixes #1 
Fixes #4

### CODE REVIEW

**Comment by nickschuch** [comment]
Solid!!!

**Comment by kimpepper** [comment]
😍

File: example/config.yml [hunk 1]
```diff
@@ -6,3 +6,43 @@ sanitize:
...
+  - webform_submission_log
+  # Non-essential data.
+  - captcha_sessions
+  - history
+  - flood
+  - batch
+  - queue
+  - sessions
+  - semaphore
+  - search_api_item
```
Comment (acbramley) [inline example/config.yml]: Just an FYI here you actually want to keep this table in particular as it contains tracking information that can only be rebuilt by disabling/reenabling the search api index and retracking everything.

Comment (nicksantamaria) [reply example/config.yml]: Right ok. Which is the table that causes Drupal to think an item is indexed, even if your local solr has no record?

**Pull request merged**